dracut/modules.d/80kiwi-verity/Makefile

```diff
@@ -0,0 +1,26 @@
+buildroot ?= /
+
+CROSS_COMPILE ?=
+CC ?= $(CROSS_COMPILE)gcc
+STRIP ?= $(CROSS_COMPILE)strip
+CFLAGS ?= -Os
+
+BINARY := kiwi-parse-verity
+
+.PHONY: install
+install: build
+	install -Dm0755 kiwi-verity-setup.sh module-setup.sh \
+		-t ${buildroot}usr/lib/dracut/modules.d/80kiwi-verity
+	install -Dm0755 $(BINARY) ${buildroot}usr/bin/$(BINARY)
+
+.PHONY: build
+build: $(BINARY)
+
+$(BINARY): kiwi-parse-verity.c Makefile
+	$(CC) -Werror -Wall -Wextra -std=c17 $(CFLAGS) -static -o $(@) $(<)
```

Comment:
Why statically linked? Most initramfs setups these days have libc and other things...

Reply:
I was thinking of what Marcus said with statically compiled tools reducing the dependencies inside the initrd when I did this, but yeah, it doesn't make much sense here when it just uses libc... I'll change that